ajax/libs/spscript/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spscript",
+  "description": "SharePoint 2013 Rest Api Wrappers",
+  "filename": "spscript.min.js",
+   "repository": {
+    "type": "git",
+    "url": "https://github.com/DroopyTersen/spscript"
+  },
+  "keywords": [
+    "sharepoint",
+    "rest",
+    "droopy",
+    "office",
+    "office-365"
+    "office365"
+  ],
+  "autoupdate": {
```

Comment:
There is no git tags on GitHub repo, so git auto-update config will not work. Let's use npm auto-update config instead.